packages/react-devtools-extensions/src/main.js

```diff
@@ -130,6 +131,7 @@ function createPanelIfReactLoaded() {
           isProfiling,
           supportsReloadAndProfile: isChrome,
           supportsProfiling,
+          supportsReact: renderers.get(1) && !!renderers.get(1).ComponentTree,
```

Comment:
Similar to above comment. I don't think this belongs as a store capability, so this can be reverted 😄